ARCHITECTURE.md

```
@@ -237,12 +266,14 @@ LLM_MAX_TOKENS=500  // Response length limit
 ## API Endpoints
 
 ### Health Check
+
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)